libraries/APM_Control/AP_PitchController.cpp

```diff
@@ -34,31 +34,31 @@ const AP_Param::GroupInfo AP_PitchController::var_info[] = {
 
 	// @Param: P
 	// @DisplayName: Proportional Gain
-	// @Description: This is the gain from pitch angle to elevator. This gain works the same way as PTCH2SRV_P in the old PID controller and can be set to the same value.
+	// @Description: Proportional gain from pitch angle demands to elevator. Higher values allow more servo response but can cause oscillations. Automatically set and adjusted by AUTOTUNE mode.
 	// @Range: 0.1 3.0
 	// @Increment: 0.1
 	// @User: User
 	AP_GROUPINFO("P",        1, AP_PitchController, gains.P,          1.0f),
 
 	// @Param: D
 	// @DisplayName: Damping Gain
-	// @Description: This is the gain from pitch rate to elevator. This adjusts the damping of the pitch control loop. It has the same effect as PTCH2SRV_D in the old PID controller and can be set to the same value, but without the spikes in servo demands. This gain helps to reduce pitching in turbulence. Some airframes such as flying wings that have poor pitch damping can benefit from increasing this gain term. This should be increased in 0.01 increments as too high a value can lead to a high frequency pitch oscillation that could overstress the airframe.
-	// @Range: 0 0.1
+	// @Description: Damping gain from pitch acceleration to elevator. Higher values reduce pitching in turbulence, but can cause oscillations. Automatically set and adjusted by AUTOTUNE mode.
+	// @Range: 0 0.2
 	// @Increment: 0.01
 	// @User: User
     AP_GROUPINFO("D",        2, AP_PitchController, gains.D,        0.04f),
 
 	// @Param: I
 	// @DisplayName: Integrator Gain
-	// @Description: This is the gain applied to the integral of pitch angle. It has the same effect as PTCH2SRV_I in the old PID controller and can be set to the same value. Increasing this gain causes the controller to trim out constant offsets between demanded and measured pitch angle.
+	// @Description: Integrator gain from long-term pitch angle offsets to elevator. Higher values "trim" out offsets faster but can cause oscillations. Automatically set and adjusted by AUTOTUNE mode.
 	// @Range: 0 0.5
 	// @Increment: 0.05
 	// @User: User
 	AP_GROUPINFO("I",        3, AP_PitchController, gains.I,        0.3f),
 
 	// @Param: RMAX_UP
 	// @DisplayName: Pitch up max rate
-	// @Description: This sets the maximum nose up pitch rate that the controller will demand (degrees/sec). Setting it to zero disables the limit.
+	// @Description: Maximum pitch up rate that the pitch controller demands (degrees/sec) in ACRO mode.
```

Comment:
how is this only an ACRO thing?

Reply:
Roll and pitch commands have always been step inputs and changes. My RMAX_RLL is 120, and it demands a roll change of 60-degree in <.05s. If it was intended to be in all modes, this is an issue.
![image](https://user-images.githubusercontent.com/35429159/50670999-ad627c80-0f94-11e9-8877-259d684926ba.png)